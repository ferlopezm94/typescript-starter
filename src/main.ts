export const greetingUppercase = (name: string): string => {
  const uppercase = `Hello ${name}!`.toUpperCase();
  console.log(uppercase);
  return uppercase;
};

console.log('test-checks');
