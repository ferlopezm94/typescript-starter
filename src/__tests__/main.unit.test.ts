import { greetingUppercase } from './../main';

describe('Main tests', () => {
  test('Should format the greeting correctly', () => {
    expect(greetingUppercase('world')).toMatchSnapshot();
  });
});
