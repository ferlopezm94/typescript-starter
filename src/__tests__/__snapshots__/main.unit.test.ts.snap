// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main tests Should format the greeting correctly 1`] = `"HELLO WORLD!"`;
